meta-mentor-staging/recipes-kernel/perf/perf.bbappend

```diff
@@ -1 +1,3 @@
 DEPENDS += "flex-native bison-native"
+REMOVE_MAN_RDEPENDS = "${@bb.utils.contains('INCOMPATIBLE_LICENSE', 'GPLv3','man', '', d)}"
```

Comment:
This won't obey incompatible license whitelisting. We handle this elsewhere in meta-mentor in a better way. See 73c743161c5873994cbb712a989d1d5c8a27a87e for example.